vizro-core/src/vizro/models/_components/form/dropdown.py

```diff
@@ -39,6 +39,34 @@ def _calculate_option_height(full_options: OptionsType) -> int:
     return 8 + 24 * number_of_lines
 
 
+def _add_select_all_option(full_options: OptionsType, multi: bool):
+    """Adds div component for select all option."""
+    if not multi:
+        return full_options
```

Comment:
```suggestion
def _add_select_all_option(full_options: OptionsType):
    """Adds div component for select all option."""
```

We could think about taking this out of the function, such that this function really only has the responsibility of adding the ALL option. 

Then you would need to call the function later on with:

` altered_options = _add_select_all_option(full_options=full_options) if self.multi else full_options`